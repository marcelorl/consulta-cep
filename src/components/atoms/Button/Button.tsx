import React from 'react'
import styled from 'styled-components'

const ButtonWrapper = styled.button`
  border-radius: 5px;
  background: ${props => props.theme.colors.cornflowerblue};
  color: ${props => props.theme.colors.white};
  font-size: 1rem;
  padding: .35rem 1rem;
`

const Button = () =>
  <ButtonWrapper>
    Buscar
  </ButtonWrapper>

export default Button
