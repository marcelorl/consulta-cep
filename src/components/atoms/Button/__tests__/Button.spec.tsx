import React from 'react'
import { shallow } from 'enzyme'

import Button from '../Button'

describe('#Button atom', () =>
  it('renders component', () => {
    const wrapper = shallow(<Button />)

    expect(wrapper).toMatchSnapshot()
  })
)
