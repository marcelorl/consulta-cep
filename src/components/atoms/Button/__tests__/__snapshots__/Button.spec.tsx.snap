// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Button atom renders component 1`] = `
<styled.button>
  Buscar
</styled.button>
`;
